frontend/components/analytics/Matomo.tsx

```diff
@@ -0,0 +1,27 @@
+import { MATOMO_URL } from '@config/env';
+import Head from 'next/head';
+
+export function Matomo() {
+   return MATOMO_URL ? (
+      <Head>
+         <script
+            key="matomo-analytics"
+            dangerouslySetInnerHTML={{
+               __html: `
+                  var _paq = window._paq = window._paq || [];
+                  /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+                  _paq.push(['trackPageView']);
+                  _paq.push(['enableLinkTracking']);
+                  (function() {
+                     var u='${MATOMO_URL}';
+                     _paq.push(['setTrackerUrl', u+'/js/tracker.php']);
+                     _paq.push(['setSiteId', '1']);
+                     var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+                     g.async=true; g.src=u+'/js/tracker.php'; s.parentNode.insertBefore(g,s);
```

Comment:
Quick check:

In production, we lightly obfuscate the tracker path (to work around basic adblockers):
https://github.com/iFixit/ifixit/blob/95a280d5a046342b69196ae04f6c082643ef3200/iFixit/Service/Matomo/MatomoPageComponent.php#L72

Reply:
@jarstelfox i think we made `minerva.php` an alias, and `tracker.js` should still work. But do we want to keep config parity there? :thinking:

Reply:
Yea that would be nice to add, I can re-qa if needed

Reply:
Can do. Will `matomo.ubreakit.com/minerva.php` work? Or should this obfuscation only be for `minerva.ifixit.com`?

Reply:
Both work, right?
https://matomo.ubreakit.com/minerva.php

Reply:
Didn't even think to follow the link myself 😅 thanks!